zerver/views/presence.py

```diff
@@ -40,7 +40,7 @@ def get_presence_backend(
     except UserProfile.DoesNotExist:
         raise JsonableError(_("No such user"))
 
-    if target.is_bot:
+    if target.bot_type is not None:
```

Comment:
Let's add a comment noting it's important to check `is_bot` here because we use `get_user_profile_narrow_by_id`, so someone doesn't clean this up improperly.